docs/experimental_notebooks/clean_poverty.ipynb

```diff
@@ -0,0 +1,194 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This cleaning file results in one combined file (poverty.csv) containing data on poverty. It is a merged version of files from SAIPE surveys conducted between the years 1998 and 2021. The original datasets were too large to be included here.     \n",
+    "\n",
+    "Data source: https://www.census.gov/programs-surveys/saipe/data/datasets.html"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "folder_pathA = 'poverty/dats'\n",
+    "folder_pathB = 'poverty'\n",
```

Comment:
This doesn't work for me locally. Please include the code in the script to pull data from the census website. You can see https://github.com/BasisResearch/cities/blob/main/docs/experimental_notebooks/membership_from_fips.ipynb as an example. 

When you're on a gov't data website, manually downloading data, you can often 'inspect' the page (right click->inspect), to get info on how to programmatically access the data. e.g. see the comment below in https://github.com/BasisResearch/cities/blob/69c6c36c6063b0c65dd7feef964696e16b7a36d8/docs/experimental_notebooks/grants_from_fips.ipynb#L23
```# to get more queries, go to USA spending website, filter what you want, then inspect page, look at network tab, and copy the request payload and header```

Reply:
I have remodeled the notebook, now it works by downloading the dataset from API.